services/queue/schemas/v1/task-group-response.yml

```diff
@@ -0,0 +1,31 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task-Group Response"
+description: |
+  Response containing information about a task group.
+type:               object
+properties:
+  taskGroupId:
+    title:          "Task-Group Identifier"
+    description: |
+      Identifier for the task-group being sealed.
+    type:           string
+    pattern:        {$const: slugid-pattern}
+  sealed:
+    type:           string
+    format:         date-time
```

Comment:
does this allow a null/empty value?

Reply:
I had struggled a bit with default null value for this, trying out different schemas here. Sometimes api validation would fail, sometimes generated GO code would complain about this. 
And I noticed that other schemas using same approach for `expired` timestamp - string type, but not required. 
So I did it same way - strict type, but field is optional. so schema allows `sealed: null` in the response. And clients didn't complain :)

Reply:
Ah, it is included in the response, even if it isn't set? Can we change that, so the response doesn't include it, if it is null?

Reply:
hm.. it's already optional, not required. Do you see a problem with this? 
I was checking other schemas how similar fields behave, and found similar approach.

We can exclude this field from response if it's null, sure